terraform/account/terraform.tfvars.json

```diff
@@ -6,23 +6,26 @@
       "is_production": false,
       "regions": [
         "eu-west-1"
-      ]
+      ],
+      "pagerduty_service_name": "OPG Modernising LPA Non-Production"
     },
     "preproduction": {
       "account_id": "792093328875",
       "account_name": "preproduction",
       "is_production": false,
       "regions": [
         "eu-west-1"
-      ]
+      ],
+      "pagerduty_service_name": "OPG Modernising LPA Non-Production"
     },
     "production": {
       "account_id": "313879017102",
       "account_name": "production",
       "is_production": true,
       "regions": [
         "eu-west-1"
-      ]
+      ],
+      "pagerduty_service_name": "OPG Modernising LPA Non-Production"
```

Comment:
Should this be:

```suggestion
      "pagerduty_service_name": "OPG Modernising LPA Production"
```

Reply:
We don't currently have a production pagerduty service to alert to, so this isn't a desired name but the name of an existing resource